File: Assignment1_IS/IS_Assign1.cpp
#include <bits/stdc++.h>
#include <iostream>
#include <string>
#define ll long long
#define lp while
#define rep(i, a, b) for (int i = a; i < b; i++)
#define sort(a) sort(a.begin(), a.end())
#define nl << endl
using namespace std;
class Assign1IS
{
    string s = "\\HelloWorld";
    public:
    int andop(char x);
    int orop(char x);
    int xorop(char x);
    void display();
};
void Assign1IS::display(){
    int len = s.length();
    rep(i,0,len){
        cout<<s[i]<<"(" << (int) s[i]<<")" nl;
        cout<<"AND: "<<andop(s[i]) nl;
        cout<<"OR: "<<orop(s[i]) nl;
        cout<<"XOR: "<<xorop(s[i]) nl;
    }
}
int Assign1IS::andop(char x){
    int a = (int) x;
    return (a&127);
}
int Assign1IS::orop(char x){
    int a = (int) x;
    return (a|127);
}
int Assign1IS::xorop(char x){
    int a = (int) x;
    return (a^127);
}

int main(){
    Assign1IS is;
    is.display();
    return 0;
}

File: IS/Assignment_1/IS_Assign1.java
public class IS_Assign1{
    int and(char x){
        return (x&127);
    }
    int or(char x){
        return (x|127);
    }
    int xor(char x){
        return (x^127);
    }
    public static void main(String[] args) {
        IS_Assign1 is = new IS_Assign1();
        String s = "\\HelloWorld";
        for(int i=0;i<s.length();i++){
            char x = s.charAt(i);
            System.out.println(s.charAt(i)+"("+(int) x+")");
            System.out.println("AND: "+ is.and(x));
            System.out.println("OR: " + is.or(x));
            System.out.println("XOR: "+is.xor(x));
        }
    }
}